speech/google/cloud/speech/_gax.py

```diff
@@ -249,8 +250,7 @@ def sync_recognize(self, sample, language_code=None, max_alternatives=None,
             raise ValueError('No results returned from the Speech API.')
 
         # Iterate over any results that came back.
-        for result in api_response.results:
-            yield Result.from_pb(result)
+        return [Result.from_pb(i) for i in api_response.results]
```

Comment:
We try to stay away from one letter vars even in list comprehensions.
`/i/result/`